Student-Unity-frontend/src/app/components/task/task.component.ts

```diff
@@ -11,39 +11,47 @@ import { STATUS } from 'src/app/models/Status';
 export class TaskComponent implements OnInit {
 
   @Input() task: Task = new Task;
-  @Output() removingDoneTaskEvent = new EventEmitter<String>();
-  @Output() removingToDoTaskEvent = new EventEmitter<String>();
-  @Output() markAsDoneEvent = new EventEmitter<String>();
-  @Output() unMarkAsDoneEvent = new EventEmitter<String>();
+  @Output() removingDoneTaskEvent = new EventEmitter<number>();
+  @Output() removingToDoTaskEvent = new EventEmitter<number>();
+  @Output() markAsDoneEvent = new EventEmitter<Task>();
+  @Output() unMarkAsDoneEvent = new EventEmitter<Task>();
 
   constructor(private studentTaskService: StudentTaskService) { }
 
   ngOnInit(): void {
   }
 
   public removeTask() {
+    this.removingDoneTaskEvent.emit(this.task.taskId);
```

Comment:
i think you forgot to delete this line, right?

Reply:
Yes I forgot to do so